test/image-prepare

```diff
@@ -46,6 +46,7 @@ def main():
     parser.add_argument('-q', '--quick', action='store_true', help='Build faster')
     parser.add_argument('-f', '--force', action='store_true', help='Force update of images')
     parser.add_argument('-b', '--build-image', action='store', help='Build in this image')
+    parser.add_argument('--devel-build-on-default-os', action='store_true', help='Do a development build on default TEST_OS')
```

Comment:
couldn't we just make this a (shorter named) flag which can work with any OS image?  this argument is practically screaming "i'm not meant to be used by any human", and i think humans might indeed sometimes want to do debug builds for local manual testing...

Reply:
Exactly, it is *not* meant for humans. A human should use `NODE_ENV=development` or `./configure --enable-debug`. I don't see a reason why a human would use this half-automagic unintuitive thing. I just found that argparse supports `help=argparse.SUPPRESS` to not show the option at all, I think I'll use that.

Reply:
I reworked this to put the logic entirely in test/run, where it IMHO fits best. This makes image-prepare and make_dist simpler also.